network/tls_wrapper.go

```diff
@@ -62,6 +64,45 @@ var (
 	tlsServerName string
 )
 
+// correct -tls_ocsp_database_url:
+// incorrect -tls_database_ocsp_url:
+
+// NamerFunc func compile final parameter name for specified service name
+type NamerFunc func(serviceName, parameterName, groupName string) string
```

Comment:
Looks like the too generic name. Maybe we can call some like `CLIParamNameConstructorFunc` or something that says that this function constructs the final name for CLI parameters for a specific service.

Reply:
agree, will rename